src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,67 @@
+package calculator;
+
+import java.util.regex.*;
+
+public class StringCalculator {
+
+    public static final int ZERO = 0;
+    public static final String DELIMITER = ",|;";
+    public static final String DELIMITER_REGEX = "//(.)\n(.*)";
+
+    public static int splitAndSum(String text) {
```

Comment:
```suggestion
    public static int sum(split 된 변수) {
```

메소드마다 하나의 책임을 가질수있도록 분리하는 방법도 좋을듯해요!